electric-motor-torque-curve/index.html

```diff
@@ -31,24 +30,84 @@ <h1>Input</h1>
       </select>
         <br><br>
 
+  <label for="no_load_torque">No load Torque (Nm):</label>
+  <input type="text" id="no_load_torque" name="no_load_torque">
+        <select>
+          <option selected="selected" value="select1">kg</option>
+          <option value = "cva2">lbs</option>
```

Comment:
Please install a HTML formmater software on your IDE with tabs set to 2 spaces (Google standard). Your tag attributes shouldn't have a space separating the = between name and value.